cli/op_error.rs

```diff
@@ -382,6 +383,21 @@ impl From<&notify::Error> for OpError {
   }
 }
 
+impl From<SwcDiagnosticBuffer> for OpError {
+  fn from(error: SwcDiagnosticBuffer) -> Self {
+    OpError::from(&error)
+  }
+}
+
+impl From<&SwcDiagnosticBuffer> for OpError {
+  fn from(error: &SwcDiagnosticBuffer) -> Self {
+    Self {
+      kind: ErrorKind::Other,
+      msg: error.diagnostics.join(", "),
+    }
+  }
+}
+
```

Comment:
@bartlomieju FYI Did this in https://github.com/denoland/deno/pull/6893/commits/4f8c2a8ea992ab9b625e1683be245084acd3f485

Reply:
Thanks for info @divy-work, I think we'll land it in this PR as it's fixing concrete test that is panicking.